Terratest/vpn/providers.tf

```diff
@@ -0,0 +1,11 @@
+provider "opentelekomcloud" {
+  alias       = "nl"
+  max_retries = 100
+  auth_url    = "https://iam.eu-nl.otc.t-systems.com/v3"
+}
+
+provider "opentelekomcloud" {

```

Comment:
explicit region and project name will be necessary for these providers. These can not be taken from env as each provider will have a different value!